package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Remove this file

Reply:
Still not removed, @habemuscode.

Reply:
Ok, I understand to remove before `npm install` and will be generated again

Reply:
I see, lock outside views/